WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -533,10 +537,7 @@ targetted_decisions = {
 						NOT = { religion_group = ROOT }
 					}
 				}
-				random = {
-					chance = 60
-					add_trait = sympathy_christendom
-				}
+				add_trait = sympathy_christendom
```

Comment:
Yes, but there is always some randomness in the game. Some people who convert are fanatic opponents against their former brothers in faith, to avoid suspicion or to signalate it is from the past.